### PR TITLE
Improve NNVM to Relay conversion

### PR DESCRIPTION
**Pull request by kazum**
This PR adds to_relay() checks to our NNVM tests.  This PR also includes some cleanups of to_relay, fixes of detected bugs, and supports of some operators to pass the test.

@jroesch @yzhliu @grwlf please help review the PR.

### CODE REVIEW

File: nnvm/python/nnvm/testing/check_computation.py [hunk 1]
```diff
@@ -441,6 +443,23 @@ def check_function(symbol, forward=None, backward=None, grad_input_vars=None,
             debug_stage = "running"
             nnvm_res = main_function(**np_inputs)
 
+            try:
+                logging.debug("checking to_relay conversion")
```
Comment (jroesch) [inline nnvm/python/nnvm/testing/check_computation.py]: Will this print every time this code is executed?

Comment (kazum) [reply nnvm/python/nnvm/testing/check_computation.py]: I think it depends on the logging configuration, but, with our CI, this message will be printed only when an error occurs.

Comment (jroesch) [reply nnvm/python/nnvm/testing/check_computation.py]: ok, seems ok

**Comment by jroesch** [comment]
Overall looks good, I could swear their were more tests written by me, but I think were moved/removed during someone's overhaul of the tests.

**Pull request merged**